/jds/bin/diff -uprN gnome-applets-2.25.90.old/null_applet/GNOME_Panel_NowApplet.server.in gnome-applets-2.25.90/null_applet/GNOME_Panel_NowApplet.server.in
--- gnome-applets-2.25.90.old/null_applet/GNOME_Panel_NowApplet.server.in	1970-01-01 01:00:00.000000000 +0100
+++ gnome-applets-2.25.90/null_applet/GNOME_Panel_NowApplet.server.in	2009-02-16 17:39:36.003044000 +0000
@@ -0,0 +1,17 @@
+<oaf_info>
+
+<oaf_server iid="OAFIID:GNOME_Panel_NowApplet"
+	    type="factory" 
+	    location="OAFIID:GNOME_NullApplet_Factory">
+
+	<oaf_attribute name="repo_ids" type="stringv">
+		<item value="IDL:GNOME/Vertigo/PanelAppletShell:1.0"/>
+		<item value="IDL:Bonobo/Control:1.0"/>
+		<item value="IDL:Bonobo/Unknown:1.0"/>
+	</oaf_attribute>
+	<oaf_attribute name="name" type="string" _value="Appointments and Tasks (Deprecated)"/>
+	<oaf_attribute name="description" type="string" _value="Sun Java System Calendar Server client applet"/>
+
+</oaf_server>
+
+</oaf_info>
diff -ruN gnome-applets-2.30.0.orig/null_applet/Makefile.am gnome-applets-2.30.0/null_applet/Makefile.am
--- gnome-applets-2.30.0.orig/null_applet/Makefile.am	2010-04-03 23:25:29.967872915 +0100
+++ gnome-applets-2.30.0/null_applet/Makefile.am	2010-04-03 23:26:23.731734212 +0100
@@ -20,6 +20,7 @@
 	GNOME_CDPlayerApplet.server.in		\
 	GNOME_MailcheckApplet_Factory.server.in	\
 	GNOME_Panel_WirelessApplet.server.in	\
+	GNOME_Panel_NowApplet.server.in		\
 	GNOME_KeyboardApplet.server.in		\
 	$(mixer_server_in)
 server_DATA = $(server_in_files:.server.in=.server)
@@ -38,6 +39,7 @@
 	GNOME_CDPlayerApplet.server.in		\
 	GNOME_MailcheckApplet_Factory.server.in	\
 	GNOME_Panel_WirelessApplet.server.in	\
+	GNOME_Panel_NowApplet.server.in		\
 	GNOME_MixerApplet.server.in		\
 	GNOME_KeyboardApplet.server.in
 
